docs/conf.py

```diff
@@ -272,10 +272,31 @@
     "https://github.com/pypa/twine/issues/*",
     # Avoid errors from channels interpreted as anchors
     "https://web.libera.chat/#",
+    # Avoid error from InvalidPyPIUploadURL docstring
+    "https://upload.pypi.org/legacy",
 ]
 
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {"https://docs.python.org/": None}
 
 # Be strict about the invalid references:
 nitpicky = True
+
+# TODO: Try to add these to intersphinx_mapping
+nitpick_ignore_regex = [
+    (r"py:.*", r"(pkginfo|requests|IO).*"),
+]
+
+# -- Options for apidoc output ------------------------------------------------
+
+autodoc_default_options = {
+    "members": True,
+    "private-members": True,
```

Comment:
I don't want us auto-documenting private members of classes we're exporting for people to use.